src/annotations/components/overlay/CreateAnnotationStreamOverlay.tsx

```diff
@@ -28,8 +28,8 @@ const DEFAULT_FLOW = JSON.stringify({
     {
       title: 'Select a Metric',
       visible: true,
-      type: 'queryBuilder',
-      ...JSON.parse(JSON.stringify(PIPE_DEFINITIONS['queryBuilder'].initial)),
+      type: 'metricSelector',
```

Comment:
RIP queryBuilder

Reply:
that's why i ended up doing this... started to migrate the DE query builder, and... the name was taken